src/csync/vio/csync_vio_local_win.cpp

```diff
@@ -133,16 +131,16 @@ std::unique_ptr<csync_file_stat_t> csync_vio_local_readdir(csync_vio_handle_t *h
           return nullptr;
       }
   }
-  auto path = QString::fromWCharArray(handle->ffd.cFileName);
+  const QString path = QString::fromWCharArray(handle->ffd.cFileName);
   if (path == QLatin1String(".") || path == QLatin1String(".."))
       return csync_vio_local_readdir(handle, vfs);
 
   file_stat.reset(new csync_file_stat_t);
-  file_stat->path = path.toUtf8();
+  file_stat->path = path;
 
-    if (vfs && vfs->statTypeVirtualFile(file_stat.get(), &handle->ffd)) {
+  if (vfs && vfs->statTypeVirtualFile(file_stat.get(), &handle->ffd)) {
```

Comment:
The indentation is more off than it used to be. Please revise and if necessary fix for the entire file once and for all.